frontend/rpm/template.go

```diff
@@ -169,6 +177,12 @@ func (w *specWrapper) Sources() (fmt.Stringer, error) {
 		}
 		fmt.Fprintf(b, "Source%d: %s\n", idx, ref)
 	}
+
```

Comment:
shouldn't `maxID` always be `len(keys)-1`?

Reply:
Yep, seems so.
Silly me.

Reply:
Fixed.